.github/workflows/ci.yml

```diff
@@ -1,8 +1,17 @@
 name: Default CI
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - 'master'
+  pull_request:
+    branches:
+      - '**'
 jobs:
-  build:
+  tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [12, 14, 16]
```

Comment:
Why run tests on all three versions?  Our intent is to support only node 16... I expect this will slow down PRs significantly, just to test against versions we want to drop support for.

Reply:
Testing on multiple versions temporarily. This will ensure that the codebase is compatible with all these versions until we totally move to Node 16 so that the MFEs running on Node 12 do not break due to components supporting Node 16 only.